src/implementations/local-paraswap-sdk.ts

```diff
@@ -99,9 +99,15 @@ export class LocalParaswapSDK implements IParaSwapSDK {
     );
 
     if (!poolPrices || poolPrices.length == 0)
-      throw new Error('Fail to get price for ' + this.dexKey);
+      throw new Error('Fail to get prices for ' + this.dexKey);
+
+    const finalPrice = poolPrices?.find(
+      p => p.prices[p.prices.length - 1] !== BigInt(MAX_UINT),
```

Comment:
In some cases we can have `0n` as well. I think we should treat that case too. I have this change in other PR, but it wasn't merge to master yet:

```
    const finalPrice = poolPrices
      // Filter pools that didn't give latest output
      .filter(poolPrices => poolPrices.prices.slice(-1)[0] !== 0n)
      .sort((a, b) => {
        return side === SwapSide.SELL
          ? Number(b.prices.slice(-1)[0] - a.prices.slice(-1)[0])
          : Number(a.prices.slice(-1)[0] - b.prices.slice(-1)[0]);
      })[0];
```

I think we can unite and add case with BI_MAX_UINT. And you don't need to convert constant numbers to `bigint`. We have them in `bigint-constants.ts`

Reply:
there you go 58be9e955d9ce9d9882702a53fcd3671f69d7c24